apps/web/pages/subscribeWithSelector.tsx

```diff
@@ -1,19 +1,24 @@
-import { useEffect } from "react";
 import create from "zustand";
-import { subscribeWithSelector } from "zustand/middleware";
```

Comment:
Does subscribeWithSelector still work with your updates?

Reply:
good question, i just assumed that since tests still do pass, everything should be fine
didn't look into it myself tho 🤔

Reply:
plust this was an empty page so i just plugged it to avoid compilation errors for the time being

Reply:
If you'd be willing to restore this manual test, that would be great!

I couldn't think of a easy way to unit test this. Something to add in the future.